.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: NPM Install
+        run: npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Run tests
+        run: |
+          npm ci
+          npm test
+  update-api-definitions-changelog:
+    runs-on: ubuntu-latest
+    needs: [test]
```

Comment:
[Q] Is it a dependent job on "test". My understanding is if "test" fails, all dependent jobs would fail too? Is there any cleanup/revert kind of jobs that we need to run irrespective of dependency failures ?

Reply:
Something like [this](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-not-requiring-dependent-jobs-to-be-successful)

Reply:
Yes, you are correct if `test` job fails all subsequent jobs should fail because it is the basic step in our release. In this case I don't think any clean up required as it is to check the test cases. If you think any clean up need to test cases please let me know we will add that. But it's a good point I might need to consider this for other jobs.